components/worktemplates/components/preview.tsx

```diff
@@ -20,10 +23,54 @@ export interface PreviewProps {
     template: WorkTemplate;
 }
 
+interface IllustrationAnimations {
+    prior: {
+        mounted: boolean;
+        type?: TemplateType;
```

Comment:
We only want to animate when the illustration change, I believe the type is unnecessary.